src/Symfony/Component/Security/Http/Controller/UserValueResolver.php

```diff
@@ -37,7 +37,7 @@ public function supports(Request $request, ArgumentMetadata $argument): bool
     {
         // with the attribute, the type can be any UserInterface implementation
         // otherwise, the type must be UserInterface
-        if (UserInterface::class !== $argument->getType() && !$argument->getAttributes(CurrentUser::class, ArgumentMetadata::IS_INSTANCEOF)) {
+        if (UserInterface::class !== $argument->getType() && !$argument->getAttributesOfType(CurrentUser::class, ArgumentMetadata::IS_INSTANCEOF)) {
```

Comment:
The `IS_INSTANCEOF` flag is no longer needed right?

Reply:
The behavior of the flag was not changed. If you pass it it allows/fetches subclasses of the attribute as well, without the flag it only matches the exact attribute name you give it. I don't know what the use case here of subclassing, but we probably shouldn't break it :)